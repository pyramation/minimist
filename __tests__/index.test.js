import { minimist } from '../src';
import cases from 'jest-in-case';

cases(
  'minmist mod',
  options => {
    expect(minimist(options)).toMatchSnapshot();
  },
  [
    ['cmd', '--dude', '1', '--bool', '-b', '-a'],
    ['cmd', 'another', '--dude', '1', '--bool', '-b', '-a'],
    ['cmd', 'one', 'more', '--dude', '1', '--bool', '-b', '-a']
  ]
);
