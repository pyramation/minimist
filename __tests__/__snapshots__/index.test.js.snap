// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first test case: 1 1`] = `
Object {
  "_": Array [
    "cmd",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;

exports[`first test case: 2 1`] = `
Object {
  "_": Array [
    "cmd",
    "another",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;

exports[`first test case: 3 1`] = `
Object {
  "_": Array [
    "cmd",
    "one",
    "more",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;

exports[`minmist mod case: 1 1`] = `
Object {
  "_": Array [
    "cmd",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;

exports[`minmist mod case: 2 1`] = `
Object {
  "_": Array [
    "cmd",
    "another",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;

exports[`minmist mod case: 3 1`] = `
Object {
  "_": Array [
    "cmd",
    "one",
    "more",
  ],
  "a": true,
  "b": true,
  "bool": true,
  "dude": 1,
}
`;
